index.html

```diff
@@ -72,6 +56,22 @@ <h1>New Entry</h1>
         </div>
       </form>
     </div>
+    <div data-view="entries" class="entries view hidden">
+      <div class="row align-center space-between">
+        <div class="column-auto">
+          <h1>Entries</h1>
+        </div>
+        <div>
```

Comment:
You should make this `column-auto` as well to keep your layout classes consistent